packages/core/types/src/workflow/order/request-item-return.ts

```diff
@@ -9,6 +10,15 @@ export interface DeleteRequestItemReturnWorkflowInput {
   action_id: string
 }
 
+export interface UpdateRequestItemReturnWorkflowInput {
+  return_id: string
+  action_id: string
+  data: {
+    quantity?: BigNumberInput
+    internal_note?: string | null
```

Comment:
question: should we be able to update return reason here?

Reply:
yes, good point.
I have to add it to the request item as well

Reply:
I will them it in a next PR because multiple files are changed...